experiment_pages/summary_ui.py

```diff
@@ -26,56 +26,85 @@ def __init__(self, input: Experiment, parent:Tk, prev_page: Frame, menu_page: Fr
         CreateExperimentButton(input, self, menu_page)
 
         scroll_canvas = ScrolledFrame(self)
-        scroll_canvas.place(relx=0.30, rely=0.25, relheight=0.7, relwidth=0.8)
+        scroll_canvas.place(relx=0.10, rely=0.25, relheight=0.7, relwidth=0.8)
 
         self.main_frame = Frame(scroll_canvas)
-        self.main_frame.grid(row=8, column=1, sticky='NESW')
-
-        self.group_frame = Frame(self.main_frame)
-        self.group_frame.grid(row=5, column=0, padx=10, pady=10, sticky='NESW')
+        self.main_frame.pack(side=LEFT, expand=True)
 
 
     def update_page(self):
-        for widget in self.group_frame.winfo_children():
+        for widget in self.main_frame.winfo_children():
             widget.destroy()
         self.create_summary_frame() 
 
 
     def create_summary_frame(self):
-        Label(self.main_frame, text=self.input.get_name()).grid(
-                row=0, column=0, padx=10, pady=10)
-        Label(self.main_frame, text=self.input.get_species()).grid(
-                row=2, column=0, padx=10, pady=10)
+        pad_x, pad_y = 10, 10
+        labels, inputs = [], []
+
+        label_style = Style()
+        label_style.configure('Summary.TLabel', font=('Arial', '10', 'bold'))
+
+        name_label = Label(self.main_frame, text='Experiment Name:', style='Summary.TLabel')
+        name_input = Label(self.main_frame, text=self.input.get_name())
+        labels.append(name_label)
+        inputs.append(name_input)
 
         names = ''
         for name in self.input.get_investigators():
-            names += name + ', '
-        Label(self.main_frame, text=names).grid(row=1, column=0, padx=10, pady=10)
+            names += name + ',\n'
+        invest_label = Label(self.main_frame, text='Investigators:', style='Summary.TLabel')
+        invest_input = Label(self.main_frame, text=names[:-2])
```

Comment:
Add a if condition to check whether the names list is > 2.

Reply:
Never expect the input is always as we need.

Reply:
the -2 is the second to last char in the investigator name string I am adding as the "," and new line. This line was put in there to remove the comma from the last item in a list. 
Would you still like a conditional?

Reply:
Yes,  by chance if get_investigators is empty then your names would be empty.

Reply:
ok done!